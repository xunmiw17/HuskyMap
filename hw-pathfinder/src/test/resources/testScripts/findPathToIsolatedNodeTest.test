CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

AddEdge simpleGraph A B 3.0
AddEdge simpleGraph B C 2.0
AddEdge simpleGraph A C 5.5

FindPath simpleGraph A D