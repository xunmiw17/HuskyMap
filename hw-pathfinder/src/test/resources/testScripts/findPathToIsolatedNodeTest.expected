created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

added edge 3.000 from A to B in simpleGraph
added edge 2.000 from B to C in simpleGraph
added edge 5.500 from A to C in simpleGraph

path from A to D:
no path found